pkg/controllers/localbuild/controller.go

```diff
@@ -151,7 +151,7 @@ func (r *LocalbuildReconciler) ReconcileArgoAppsWithGitea(ctx context.Context, r
 	}
 	r.shouldShutdown = shutdown
 
-	return ctrl.Result{RequeueAfter: time.Second * 30}, nil
+	return ctrl.Result{RequeueAfter: time.Second * 15}, nil
```

Comment:
is there thoughts as to why we changed this? Is there ratelimiting concerns with a lower value?

Reply:
Not really lol. I just wanted to the tests to finish faster. TBH the 30 second  requeue time is also a number I made up when I was writing this first.

Reply:
I'd say lets revert it back to 30.